stl/inc/functional

```diff
@@ -1139,12 +1139,37 @@ public:
 _NON_MEMBER_CALL(_FUNCTION_POINTER_DEDUCTION_GUIDE, X1, X2, X3)
 #undef _FUNCTION_POINTER_DEDUCTION_GUIDE
 
+template <class _Fx, class _Call_op, class = void>
+struct _Deduce_from_call_operator : _Is_memfunptr<_Call_op>::_Guide_type {}; // N4958 [func.wrap.func.con]/16.1
+
+#ifdef __cpp_static_call_operator // TRANSITION, P1169R4

```

Comment:
I wonder whether this should be `TRANSITION`...

Clang already backports `static` `operator()` to all old modes ([doc](https://clang.llvm.org/docs/LanguageExtensions.html#language-extensions-back-ported-to-previous-standards), [demo](https://godbolt.org/z/5Yzjohf53)). I don't know whether MSVC & EDG will do the same.
If `static` `operator()` (along with explicit `this` parameter) is backported to C++17/20 modes, IMO corresponding deduction guides should also be available (like the library support for `char8_t`).

Reply:
Interesting, I didn't know that! I agree that the guides should be guarded by the feature-test macro without a TRANSITION comment.